### DISCUSSION TITLE
preserve mute state across chained patterns ?

### DISCUSSION THREAD

**Discussion by enorrmann** [bangcorrupt/hacktribe#300] [Ideas]
this could improve jamming possibilities, ie, muting drums and letting a long melody run across several patterns, then unmute drums and such
currently mute state is saved with the pattern

**Comment by bangcorrupt** [comment]
I like this idea.  A switch to select either global or pattern mutes would be good too.

**Comment by vanana-old-acc** [comment]
Great idea that would make a big difference for live jamming
🥇
